pkg/xen-tools/Dockerfile

```diff
@@ -3,7 +3,7 @@ RUN apk add --no-cache \
     	    gcc make libc-dev dev86 xz-dev perl bash python-dev \
 	    gettext iasl util-linux-dev ncurses-dev glib-dev \
 	    pixman-dev libaio-dev yajl-dev argp-standalone \
-	    linux-headers git patch texinfo curl tar
+	    linux-headers git patch texinfo curl tar libcap-dev attr-dev
```

Comment:
yetus seems to be complaining about tabs vs. spaces here. Can you please check?

Reply:
fixed.